### ISSUE TITLE
figure do not show up in treatment

### ISSUE THREAD

**Issue by myrmoteras** [gbif/portal-feedback#3294]
might it be possible to check this https://www.gbif.org/species/179911497  

![image](https://user-images.githubusercontent.com/4609956/109932224-3509ba00-7cca-11eb-94e0-84c7b26ecf38.png)

http://treatment.plazi.org/GgServer/html/7209878F844BFFD6FC8EFE7EFE9D28EE

this should include figures - and it is part of my today's lecture and thus would be nice to have the images in place...

tx for looking into  this
d

**Comment by MattBlissett** [comment]
Hi Donat,

We had checklist crawling switched off in order to prepare a new GBIF backbone.  A small issue with the new backbone required us to regenerate it, but with about 150 checklists in the queue we turned crawling back on for a short time, let the queued checklists be processed, then turned crawling off again.

This treatment was first published without the multimedia, and that is the version which we crawled.  Unfortunately, we are in the process of switching to the new backbone, and it would be difficult to recrawl this dataset before the new backbone is released.

We'll discuss whether there are ways to improve our processing to reduce the time checklist crawling is suspended.

Thanks,
